### ISSUE TITLE
Kill Streak Perks

### ISSUE THREAD

**Issue by punxtr** [BitLimit/PvP#9]
Once attaining a kill streak of 10+ you will be rewarded a more powerful sword. Once attaining a kill streak of 20+ you will be awarded a more powerful bow. It shouldn't get anymore OP than that. Never put Knockback on the upgraded sword.

**Comment by afomera** [comment]
Interesting. I wonder how this would play out.... What kind of enchantments would you want to see from this? 

I also think maybe people should get more food if they get more kills due to the fact killed players don't drop items anymore...

**Comment by punxtr** [comment]
Indeed. Increased food is a must, especially for lone wolves that can't have shared supplies.

The enchants will be Sharpness IIs and Power IIs. Then increase. Nothing too great.

**Comment by kolinkrewinkel** [comment]
One idea I had would be that you could spend your kill exp on an enchanted item (spawn an iron sword server side, enchant, give to player, take levels.)
